src/averageAges.js

```diff
@@ -55,7 +71,21 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const child = onlyWithSon
+    ? people.filter(person => people.find(
+      mother => person.mother === mother.name
+    ) && person.sex === 'm')
+    : people.filter(person => people.find(
+      mother => person.mother === mother.name
+    ));
```

Comment:
please pay attention to spaces
```suggestion
  const child = onlyWithSon
    ? people.filter(person => people.find(
        mother => person.mother === mother.name
      ) && person.sex === 'm')
    : people.filter(person => people.find(
        mother => person.mother === mother.name
      ));
```